core/src/mindustry/game/SectorInfo.java

```diff
@@ -116,7 +118,7 @@ public void handleItemExport(Item item, int amount){
 
     /** Subtracts from export statistics. */
     public void handleItemImport(Item item, int amount){
-        export.get(item, ExportStat::new).counter -= amount;
+        //import statistics are handled separately
```

Comment:
Why is this here?

Reply:
As far as I'm aware, `handleItemImport` isn't used anywhere at the moment. I removed that line so that any future calls won't accidentally change export statistics when it should change import statistics. But maybe I misunderstood something here